### PR TITLE
Fix the search in the agent inventory data tables

### PR DESCRIPTION
**Pull request by Desvelao**
### Description
This pull request changes the query parameter used to search in the agent inventory tables.
 
### Issues Resolved
https://github.com/wazuh/support/issues/171

### Evidence

image
image
image

### Test

UI
| Test |
| --- |
| Empty input in the search bar of the agent inventory tables should display all the results |
| Add a search text input in the search bar of the agent inventory tables should display filtered the results |
| Export to CSV each table in agents inventory with an empty search query should display all the results |
| Export to CSV each table in agents inventory with a search query should display results filtered |
| The search bar placeholder of Network ports should be Search |
| The search bar placeholder of Packages should be Search |
| The search bar placeholder of Processes ports should be Search |

[template_test_cases.md](https://github.com/wazuh/wazuh-kibana-app/files/10667603/tests.4.md)

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by Machi3mfl — APPROVED** [review]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Empty input in the search bar of the agent inventory tables should display all the results | :green_circle: | :black_circle: | :black_circle: |
| Add a search text input in the search bar of the agent inventory tables should display filtered the results | :green_circle: | :black_circle: | :black_circle: |
| Export to CSV each table in agents inventory with an empty search query should display all the results | :green_circle: | :black_circle: | :black_circle: |
| Export to CSV each table in agents inventory with a search query should display results filtered | :green_circle: | :black_circle: | :black_circle: |

**Details**
<details>
<summary>:green_circle: Empty input in the search bar of the agent inventory tables should display all the results</summary>

Chrome - :green_circle:

https://user-images.githubusercontent.com/6089438/217066932-c44c3110-1bb4-4eb2-8059-d848c4054649.mp4

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Add a search text input in the search bar of the agent inventory tables should display filtered the results</summary>

Chrome - :green_circle:

https://user-images.githubusercontent.com/6089438/217067433-ef759b9f-7870-4110-88d9-a10472d0ffce.mp4

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Export to CSV each table in agents inventory with an empty search query should display all the results</summary>

Chrome - :green_circle:

https://user-images.githubusercontent.com/6089438/217068030-8a777f5e-b79b-4e1c-b995-7274a46d6382.mp4

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Export to CSV each table in agents inventory with a search query should display results filtered</summary>

Chrome - :green_circle:

https://user-images.githubusercontent.com/6089438/217068094-0a40d07f-7700-444d-9ec1-9564a6c70c9c.mp4

Firefox - :black_circle:

Safari - :black_circle:

</details>

**Comment by yenienserrano** [comment]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Empty input in the search bar of the agent inventory tables should display all the results | :black_circle: | :black_circle: | :black_circle: |
| Add a search text input in the search bar of the agent inventory tables should display filtered the results | :black_circle: | :black_circle: | :black_circle: |
| Export to CSV each table in agents inventory with an empty search query should display all the results | :black_circle: | :black_circle: | :black_circle: |
| Export to CSV each table in agents inventory with a search query should display results filtered | :black_circle: | :black_circle: | :black_circle: |

**Details**
<details>
<summary>:black_circle: Empty input in the search bar of the agent inventory tables should display all the results</summary>

Chrome - :black_circle:

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:black_circle: Add a search text input in the search bar of the agent inventory tables should display filtered the results</summary>

Chrome - :black_circle:

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:black_circle: Export to CSV each table in agents inventory with an empty search query should display all the results</summary>

Chrome - :black_circle:

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:black_circle: Export to CSV each table in agents inventory with a search query should display results filtered</summary>

Chrome - :black_circle:

Firefox - :black_circle:

Safari - :black_circle:

</details>

**Review by yenienserrano — APPROVED** [review]

**Comment by Desvelao** [comment]
**request changes**: change the placeholder of the search bar to clarify the search is only not to a column. The search applies to all the visible columns.

**Review by Machi3mfl — APPROVED** [review]
|||
|---|---|
| The search bar placeholder of Network ports should be Search | :green_circle:  |
| The search bar placeholder of Packages should be Search | :green_circle: |
| The search bar placeholder of Processes ports should be Search | :green_circle: |

**Details**

<details>
<summary> :green_circle: The search bar placeholder of Network ports should be Search</summary>

image

</details>

<details>
<summary> :green_circle: The search bar placeholder of Packages should be Search</summary>

image

</details>

<details>
<summary> :green_circle: The search bar placeholder of Processes ports should be Search</summary>

image

</details>

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.4.1 - OpenSearch Dashboards 2.6.0 - Revision 07
+
+### Change
+
+- Change the search method in the agents inventory tables [#5196](https://github.com/wazuh/wazuh-kibana-app/pull/5196)
```
Comment (AlexRuiz7) [inline CHANGELOG.md:9]: Move to `Fixed`

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Desvelao) [reply CHANGELOG.md]: Fixed here: https://github.com/wazuh/wazuh-kibana-app/pull/5196/commits/4f8d376e050ab15ed5d9a80ee2b964337f6e7a07

**Review by AlexRuiz7 — DISMISSED** [review]

**Review by Machi3mfl — APPROVED** [review]

**Review by AlexRuiz7 — DISMISSED** [review]

**Review by asteriscos — APPROVED** [review]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | 
| --- |  --- | --- |
| Empty input in the search bar of the agent inventory tables should display all the results | :green_circle: | :black_circle: | 
| Add a search text input in the search bar of the agent inventory tables should display filtered the results | :black_circle: | :black_circle: | 
| Export to CSV each table in agents inventory with an empty search query should display all the results | :black_circle: | :black_circle: | 
| Export to CSV each table in agents inventory with a search query should display results filtered | :black_circle: | :black_circle: | 

**Details**
<details>
<summary>:green_circle: Empty input in the search bar of the agent inventory tables should display all the results</summary>

Chrome - :green_circle:

Firefox - :black_circle:

Screenshot from 2023-03-30 16-27-19


</details>

<details>
<summary>:green_circle: Add a search text input in the search bar of the agent inventory tables should display filtered the results</summary>

Chrome - :green_circle:

Firefox - :black_circle:

image


</details>

<details>
<summary>:green_circle: Export to CSV each table in agents inventory with an empty search query should display all the results</summary>

Chrome - :green_circle:

Firefox - :black_circle:

Screenshot from 2023-03-30 16-56-22
Screenshot from 2023-03-30 16-56-10


</details>

<details>
<summary>:green_circle: Export to CSV each table in agents inventory with a search query should display results filtered</summary>

Chrome - :green_circle:

Firefox - :black_circle:

Screenshot from 2023-03-30 16-56-16
Screenshot from 2023-03-30 16-56-05


</details>

**Review by yenienserrano — APPROVED** [review]

**Pull request merged**